modules/performance/extension/siddhi-execution-performance/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.wso2.sp</groupId>
         <artifactId>wso2sp-performance</artifactId>
-        <version>4.4.0-m2-SNAPSHOT</version>
+        <version>4.4.0-m3-SNAPSHOT</version>
```

Comment:
No need to change the version

Reply:
This has to be changed since there is a mismatch between the parent pom file version which is 4.4.0-m3-SNAPSHOT.